lib/genre.rb

```diff
@@ -0,0 +1,34 @@
+class Genre
+  extend Concerns::Findable
+
+  attr_accessor :name
+  attr_accessor :songs
```

Comment:
#### Code Quality - Nitpick
All the ```attr_accessor```s should be put on the same line, separated by commas like so:

```attr_accessor :name, :songs```